modules/java/proxies/java-destination-proxy.c

```diff
@@ -145,6 +145,12 @@ __load_destination_object(JavaDestinationProxy *self, const gchar *class_name, c
                                                    self->dest_impl.mi_constructor, handle);
   if (!self->dest_impl.dest_object)
     {
+      jthrowable exc = CALL_JAVA_FUNCTION_VOID(java_env, ExceptionOccurred);
+      if (exc)
+        {
+          CALL_JAVA_FUNCTION_VOID(java_env, ExceptionDescribe);
```

Comment:
I was experimenting with that, but in the end I could not find a solution after putting a small effort into it.

I found a function call (I don't remember now which was that, but I can regoogle that if interesting) that could extract the error message from the exception, that needed to be converted to utf8 string through java (it was a little bit complex code), and also the result was not satisfactory, because the stacktrace part was missing.

In my case the exception came from the framework itself (missing jar files from make install), and not triggered from user code. Maybe this could be triggered from user code, but users can write their code with appropriate logging that they do not need to rely on us for do so. That's why I did not add to much importance to the matter.

So I left this change in the patchset as good enough. But of course it can probably improved into an msg_error() message later.

Reply:
I do not think this is mandatory as; there is a general error about a missing constructor via internal log, so the user could be notified that something is wrong; this just gives an extra help for finding it out.
Maybe this could be moved to debug level; but I am fine with it as it is.